research-software-directory/RSD-as-a-service
# fix: remove backticks in the markdown content and style text

# Markdown formatting

Closes #899

Changes proposed in this pull request:
*  Disabled backtics produced by tailwind/typography plugin (css)
*  Added code style for inline code to "mimick" github styling when backticks are used   

How to test:
* `make start` to build app and generate test data
*  login to RSD
* create new software. Use [this markdown as example](https://raw.githubusercontent.com/tpronk/splithalfr/main/README.md)
* It should not have backticks in bulletpoints and should look quite similair to [github version](https://github.com/tpronk/splithalfr#how-to-cite)
* try using with direct url linking. It should produce similar result
* try markdown feature with a new project markdown content. It should work too.
* try same with organisation about page. It should work too.

## Example software markdown
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/c6ee40ea-d54c-4d11-9d9e-12d87402f4bc)


PR Checklist:
